src/components/Professor/ProfessorSearch/style.scss

```
@@ -0,0 +1,25 @@
+@use '@/style/package' as *;
+
+.searchBar {
+  margin-bottom: 8px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  input {
+    border: none;
+    width: 15rem;
+    font-size: 16px;
+    background: bottom;
+    color: $color-white;
+    text-align: center;
+  }
+  input::placeholder {
+    color: $color-white;
+  }
```

```suggestion
    text-align: center;
 
    &::placeholder {
      color: $color-white;
    }
  }
```

nesting rule 을 적용해봣어

---

완!